### PR TITLE
Give a small example of a file+noindex:// repo layout

### PR DESCRIPTION
**Pull request by bubba**
---
Please include the following checklist in your PR:

* [x] Patches conform to the [coding conventions](https://github.com/haskell/cabal/blob/master/CONTRIBUTING.md#conventions).
* [x] Any changes that could be relevant to users have been recorded in the changelog (add file to `changelog.d` directory).
* [x] The documentation has been updated, if necessary.

Please also shortly describe how you tested your change. Bonus points for added tests!

### CODE REVIEW

File: Cabal/doc/installing-packages.rst [hunk 1]
```diff
@@ -151,7 +151,14 @@ repository.
     repository my-local-repository
       url: file+noindex:///absolute/path/to/directory
 
-``cabal`` will construct the index automatically from the
+Given that ``/absolute/path/to/directory`` looks like
+::
+
+    /absolute/path/to/directory/
+        foo-0.1.0.0.tar.gz
+        bar-0.2.0.0.tar.gz
```
Comment (phadej) [inline Cabal/doc/installing-packages.rst:159]: I would
- also add `bar-0.2.0.0.cabal` file 
- reorder this to first say what was already i.e. *`cabal`` will ...`, and then
- add:

For example from the directory contents

```
foo-0.1.0.0.tar.gz
bar-0.2.0.0.tar.gz
bar-0.2.0.0.cabal
```

`cabal` will create an index with two packages, `foo-0.1.0.0` and `bar-0.2.0.0` using in-tarball `.cabal` file for `foo` and `bar-0.2.0.0.cabal` filefor foo.

TL;DR first tell what, then show an example. We should avoid examples as part of "specification".

**Comment by phadej** [comment]
Thanks! Merged as https://github.com/haskell/cabal/commit/3150547fb07e5983a0c7a3625cccf0cbeeb46baa
